src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -459,9 +460,6 @@ - (void)writeAttributeWithEndpointID:(NSNumber *)endpointID
                        timedWriteTimeout:timeout
                              clientQueue:self.queue
                               completion:^(NSArray<NSDictionary<NSString *, id> *> * _Nullable values, NSError * _Nullable error) {
-                                  if (values) {
-                                      [self _handleAttributeReport:values];
```

Comment:
This seems fine, but should we trigger an immediate revert of the temp value this write added if we errored out?

Reply:
File #23620 - will address that separately.